docs/guides/hooks-list.rst

```diff
@@ -447,6 +447,9 @@ Other
 **robots.txt, site**
 	Filter the robots.txt values for ``$params['site']``.
 
+**config, amd**
+	Change the AMD config for the requirejs library.
```

Comment:
`'config', 'htmlawed'` is also used, and it changes the **config** of AMD, and not a filter...